platform-sdk/swirlds-base/src/test/java/com/swirlds/base/context/GlobalContextTest.java

```diff
@@ -17,28 +17,41 @@
 package com.swirlds.base.context;
 
 import com.swirlds.base.context.internal.GlobalContext;
-import com.swirlds.base.test.fixtures.context.WithContext;
+import com.swirlds.base.context.internal.ThreadLocalContext;
 import java.util.Map;
+import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
-@WithContext
 public class GlobalContextTest {
```

Comment:
This test should use @WithContext instead of the manual work in the setUp/tearDown method. in the maven project that was not possible based on missing textFixtures but here it should work